packages/scripts/src/get-changed-adapters/soakTestBlacklist.ts

```diff
@@ -33,6 +33,7 @@ export const SoakTestBlacklist: string[] = [
   'expert-car-broker',
   'flightaware',
   'fmpcloud',
+  'frxeth-exchange-rate',
```

Comment:
No soak tests for this one?

Reply:
I'll enable them in a follow up once I've updated the adapter secrets!